middlewares/account.middleware.js

```diff
@@ -159,8 +159,16 @@ async function addAccount(req, res, next) {
  * @param {*} next 
  */
 async function updateAccount(req, res, next) {
-    const account = await Services.Account.updateOne(req.params.id, req.body);
-    if (account) {
+    var account = await Services.Account.findById(req.params.id)
+
+    // If we are changing the email, and there is a difference between the two, set back to unconfirmed status.
+    if (req.body.email && account.email != req.body.email) {
```

Comment:
why are we checking for both? is there a way we can not call this if `req.body.email` is unchanged?

Reply:
Currently, if a field isn't given to the request then it doesn't change it. So if we don't check if req.body.email is undefined then if somebody has an email and they change their name, it'll make them unconfirmed.

Not sure of a better way

Reply:
Would the PR for parsePatch break this check?

Reply:
Yes! Talked to @Ty-Won and he said to add TODO as his patch is currently breaking, for when this hits.